Manifests/ManagedPreferencesApplications/com.vpntracker.365mac-config.plist

```diff
@@ -0,0 +1,344 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>pfm_app_url</key>
+	<string>https://www.vpntracker.com</string>
+	<key>pfm_description</key>
+	<string>Manage settings in VPN Tracker for Mac, iPhone or iPad – the multi protocol VPN client for enterprise teams. Visit vpntracker.com/goto/profile to learn more.</string>
+	<key>pfm_domain</key>
+	<string>com.vpntracker.365mac</string>
+	<key>pfm_format_version</key>
+	<integer>6</integer>
+	<key>pfm_icon</key>
+	<data>
```

Comment:
This _shouldn't_ be necessary.  Will auto load an icon of the same name as the manifest in `Icons/ManagedPreferenfcesApplications/<name_of_manifest.png`.  Should be 64x64

Reply:
Thank you, will provide an icon file with our next push.